### DISCUSSION TITLE
Roles based login

### DISCUSSION THREAD

**Discussion by naveenprasathselvam** [filamentphp/filament#1283] [General]
Hi, Please help me how to do with roles based login

**Comment by ManojKiranA** [comment]
what you mean by roles based login you mean multi guard or multi roles ?

**Comment by ehs4nm** [comment]
I think this question could be:
How to implement role base panels, how to create client panel with separated route and separate panel?
I have the same question.
I need to provide my client a panel with parts of admin panel in it. i want it to have prefix route like: ../clients/dashboard
and i don't want to do it in jetstream or other package and repeat all the stuff i have created for admin panel.

**Comment by danharrin** [comment]
I suggest using policies to control access to your resources based on user type. https://filamentphp.com/docs/2.x/admin/resources#authorization
